oil_lang/objects.py

```diff
@@ -33,34 +35,35 @@ def __getitem__(self, typ):
     raise AssertionError('typ: %s' % typ)
 
   def __call__(self):
+    # type: () -> None
     # Array(1 2 3)
     raise AssertionError("Arrays need a parameterized type")
 
 
 # These are for data frames?
 
-class BoolArray(list):
+class BoolArray(List[bool]):
```

Comment:
I'm not positive but I think this will cause a runtime error with `vendor/typing.py`?  i.e. I think `bin/osh` imports that.

Note the PYTHONPATH in `bin/osh`:

```
PYTHONPATH=$REPO_ROOT:$REPO_ROOT/vendor exec $REPO_ROOT/bin/oil.py osh "$@"
```

Reply:
Yep, that was broken too.  Fixed by: 02b1a4fa1162d1a18007acb62d5ee5d05b114f3f